### ISSUE TITLE
[Automatic Installation] How to uninstall the mod?

### ISSUE THREAD

**Issue by jeroentyx** [MrBacanudo/SkulHardModeMods#3]
Hi
 I downloaded the Mod using the automatic installation. I only found a executable file in my downloads folder.
Could you guide me to uninstall the mod? because I don't know what are the files to remove from the skul folder
Thank you

**Comment by MrBacanudo** [comment]
Sorry about the delay, @jeroentyx!

To temporarily disable the mods, you can just edit the `doorstop_config.ini` file on the Skul Folder to say `enabled=false` instead of `enabled=true`.

To uninstall everything, delete the folders and files from the Skul folder:
- BepInEx
- 2020.3.34
- doorstop_config.ini
- winhttp.dll

**Comment by Det3c** [comment]
hello sir i want to ask, im abit confuse here how to find the folder 2020.3.34 in my skull folder i cant find it and the same like BepInEx in guide manuall instalation i need to put mod in "BepInEx/plugins" but i cant find plugins file should i need to create both file the folder called 2020.3.34 and plugins and last one where extract these file too "Unstripped Core Libraries" & "Unstripped Unity Libraries" thanks sorry for bothering you

**Comment by MrBacanudo** [comment]
@Det3c, yes, you need to create those folders, then add the files inside them.
